compiler/src/main/kotlin/dev/zacsweers/metro/compiler/fir/ExtensionPredicates.kt

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2025 Zac Sweers
+// SPDX-License-Identifier: Apache-2.0
+package dev.zacsweers.metro.compiler.fir
+
+import dev.zacsweers.metro.compiler.ClassIds
+import dev.zacsweers.metro.compiler.asFqNames
+import org.jetbrains.kotlin.fir.extensions.predicate.DeclarationPredicate
+import org.jetbrains.kotlin.fir.extensions.predicate.LookupPredicate
+
+public class ExtensionPredicates(private val classIds: ClassIds) {
```

Comment:
nice. Let's keep these internal though!

```suggestion
internal class ExtensionPredicates(private val classIds: ClassIds) {
```